README.md

```diff
@@ -104,8 +105,11 @@ before issuing a pull request on GitHub.
 make style
 ----------
 
-Run the PSR2 and PEP8 style checks on MVC PHP code and Python,
+Run the PSR12 and PEP8 style checks on MVC PHP code and Python,
 respectively.
```

Comment:
it should probably be noted that the "os-debug" or "os-debug-devel" plugin will install all required dependencies.

Reply:
What that tool, I can not find any references about in on the web ?
links could be put in the README